features/auth/LoginScreen.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import { Alert, Image, StyleSheet, Text, View, KeyboardAvoidingView } from 'react-native';
+import AppInput from '../../components/AppInput'
+import AuthButton from '../../components/AuthButton';
+import auth from '@react-native-firebase/auth';
+
+const LoginScreen = ({ navigation }) => {
```

Comment:
passing the navigation as props is not needed once you create the RootNavigation